packages/icons/src/svg/data-fabric-colored.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+  <path class="ti-data-fabric-colored-bg" fill="#0675C1" d="M16.0001,7.9998 C16.0001,12.4178 12.4181,15.9998 8.0001,15.9998 C3.5821,15.9998 0.0001,12.4178 0.0001,7.9998 C0.0001,3.5818 3.5821,-0.0002 8.0001,-0.0002 C12.4181,-0.0002 16.0001,3.5818 16.0001,7.9998"/>
```

Comment:
per our previous dicussion all element of an icon should be wrapped into a root element so filters could be applied on IE11 to.

Reply:
Good one ! This one is quite urgent on Portal, once VD's get a CSS example code, we can from then on implement it in a root element